backend/app/services/storage/minio_client.py

```
@@ -0,0 +1,213 @@
+        bucket_name = settings.MINIO_BUCKET_NAME
+
+        try:
+            response = self.client.get_object(
+                bucket_name=bucket_name,
+                object_name=object_key,
+            )
+            data = response.read()
+            response.close()
+            response.release_conn()
+            logger.debug(f"Downloaded file from MinIO: {bucket_name}/{object_key} ({len(data)} bytes)")
+            return data
```

Ruff (0.14.7)

Consider moving this statement to an `else` block

(TRY300)